### ISSUE TITLE
[HOLD https://github.com/Expensify/App/pull/20512 $1000] Web - New tooltip is not displayed over workspace direct chats and general workspace chat name

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#20716]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
**Issue found when executing PR** https://github.com/Expensify/App/pull/20276
## Action Performed:
1. Login to NewDot with account with workspace with several members
2. Hover over workspace direct chat user names and icons (and over general workspace chat)

## Expected Result:
New tooltip should be displayed

## Actual Result:
Tooltip is missing

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.27.4

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/e7506f18-2031-4077-aab1-87d3a7ecfbaf

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01443ce644ff2fc4f2</li>
        <li>Upwork Job ID: 1671866330621509632</li>
        <li>Last Price Increase: 2023-06-22</li>
    </ul>
</details>

**Comment by isagoico** [comment]
Issue found while executing PR https://github.com/Expensify/App/pull/20276

**Comment by iwiznia** [comment]
@puneetlath @fedirjh is this something we missed on https://github.com/Expensify/App/pull/20276 or is this how it is supposed to work? I assume the former...

**Comment by iwiznia** [comment]
Yeah, I don't see any mention in the OG issue of other entries in the LHN that are not DMs... but our LHN has a lot of things that are not DMs. So the question is: do we want to show a tooltip for all the other cases or not? I'd assume we do want to do that, though not sure what the tooltip would show.

**Comment by muttmuure** [comment]
- Are we sure that this isn't restricted to only the blank unread chats that appear when someone looks you up on New Expensify? 
- If it is that, I think we plan to remove that behavior. Therefore it might seem silly to fix a bug with something that might not be there. 
-  If this is something more broad, as in all DMs in Workspaces, I agree this is a bug that we should fix.

**Comment by iwiznia** [comment]
> Are we sure that this isn't restricted to only the blank unread chats that appear when someone looks you up on New Expensify?

I don't get this... can you rephrase? What is a blank unread chat?

> If this is something more broad, as in all DMs in Workspaces, I agree this is a bug that we should fix.

Very confused by this... the tooltip is not showing for workspace chats, that's the bug.

**Comment by muttmuure** [comment]
> I don't get this... can you rephrase? What is a blank unread chat?

When someone searches your name on New Expensify and clicks your name in Search, a chat with them appears in **your** LHN

**Comment by muttmuure** [comment]
My question is: 

- Does this affect those kinds of chats I have just described above or does it affect all workspace chats?

It sounds like you are confirming that it affects all workspace chats

**Comment by iwiznia** [comment]
> Does this affect those kinds of chats I have just described above or does it affect all workspace chats?

I am confused because I don't think that happens in workspace chats? But it's much easier for you to do the tests than to try to explain them to me and me back to you.

**Comment by muttmuure** [comment]
@stitesExpensify ready to assign @hoangzinh

**Comment by stitesExpensify** [comment]
Added a hold for https://github.com/Expensify/App/pull/20512. When that is merged we can come back and see if this is still happening!

**Comment by stitesExpensify** [comment]
hold

**Comment by stitesExpensify** [comment]
This is the intended functionality right? 
<img width="372" alt="2023-07-21_12-52-38" src="https://github.com/Expensify/App/assets/42391420/2d451761-376c-4f59-adc2-c852b5f78d50">

**Comment by stitesExpensify** [comment]
Cool, then this is fixed!
